helm/conjur-app-deploy/charts/app-secretless-broker/templates/test_app_secretless_broker.yaml

```diff
@@ -87,11 +87,11 @@ spec:
       {{- if eq .Values.app.platform "kubernetes" }}
       imagePullSecrets:
         - name: dockerpullsecret
-      {{- end }}
       securityContext:
         fsGroup: 65534
         runAsGroup: 65534
         runAsUser: 65534
+      {{- end }}
```

Comment:
Ooof, thanks for fixing this. I forgot OpenShift does its own security context thing.